### PR TITLE
Create 2023-12-07-semantic-options-benchmarks.md

### PR DESCRIPTION
**Pull request by sean-zheng-amazon**
### Description
Publish blog: Semantic Search with OpenSearch: Architecture Options and Benchmarks
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by pajuric** [comment]
@nateynateynate @dtaivpp @krisfreedain - Blog is approved and ready to publish today.

File: _posts/2023-12-07-semantic-options-benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "Semantic Search with OpenSearch: Architecture options and Benchmarks"
+authors:
+ - sean-zheng-amazon
```
Comment (krisfreedain) [inline _posts/2023-12-07-semantic-options-benchmarks.md:5]: 'sean-zheng-amazon' author profile does not exist - should it be 'seanzheng'? Please confirm/fix

File: _posts/2023-12-07-semantic-options-benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+ - zane-neo
```
Comment (krisfreedain) [inline _posts/2023-12-07-semantic-options-benchmarks.md:6]: 'zane-neo' author profile does not exist - who should this be? Please confirm/fix

File: _posts/2023-12-07-semantic-options-benchmarks.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+ - ylwu-amzn
```
Comment (krisfreedain) [inline _posts/2023-12-07-semantic-options-benchmarks.md:7]: 'ylwu-amzn' author profile does not exist - should it be 'ylwu'? Please confirm/fix

**Review by krisfreedain — DISMISSED** [review]
Author profiles are wrong - please resolve

File: _posts/2023-12-07-semantic-options-benchmarks.md (hunk 1 above)
Comment (nateynateynate) [inline _posts/2023-12-07-semantic-options-benchmarks.md:5]: None of the authors are appearing. Are there entries in _authors with a matching 'short_name' ?

File: _posts/2023-12-07-semantic-options-benchmarks.md (hunk 2 above)
Comment (nateynateynate) [inline _posts/2023-12-07-semantic-options-benchmarks.md:6]: Not seeing an author short_name defined in _authors for this author either.

File: _posts/2023-12-07-semantic-options-benchmarks.md (hunk 3 above)
Comment (nateynateynate) [inline _posts/2023-12-07-semantic-options-benchmarks.md:7]: Yaliang's shortname is `ylwu`.

**Review by nateynateynate — CHANGES_REQUESTED** [review]
Try using 'bundle exec jekyll serve' from the CLI and then visit localhost:4000 -  you can see your blog beore you submit the PR. 

If we can address these author name issues I'll get this through as soon as I can.

File: _posts/2023-12-07-semantic-options-benchmarks.md (hunk 2 above)
Comment (sean-zheng-amazon) [reply _posts/2023-12-07-semantic-options-benchmarks.md]: added zan's bio and pic

File: _posts/2023-12-07-semantic-options-benchmarks.md (hunk 3 above)
Comment (sean-zheng-amazon) [reply _posts/2023-12-07-semantic-options-benchmarks.md]: changed to ylwu

File: _posts/2023-12-07-semantic-options-benchmarks.md (hunk 1 above)
Comment (sean-zheng-amazon) [reply _posts/2023-12-07-semantic-options-benchmarks.md]: yes, changed to 'seanzheng'

**Review by nateynateynate — APPROVED** [review]
Looks good locally here. Let's publish!

**Pull request merged**